src/main/modules/helmet/modules/contentSecurityPolicy.ts

```diff
@@ -9,7 +9,7 @@ export class ContentSecurityPolicy {
   constructor (public developmentMode: boolean) {}
 
   enableFor (app: express.Express) {
-    const scriptSrc = [self, '*.google-analytics.com']
+    const scriptSrc = ['\'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU=\'', '\'sha256-G29/qSW/JHHANtFhlrZVDZW1HOkCDRc78ggbqwwIJ2g=\'', self, '*.google-analytics.com']
```

Comment:
Perhaps it would be good to add a comment here which tells which inline scripts these hashes refer to?

Reply:
ah true. I should extract them to meaningful variables. will do so now

Reply:
have pushed some changes